integration/runner/runner.go

```diff
@@ -79,7 +104,7 @@ func PushAndRunTests(host, testDir string) error {
 	portStr := strconv.Itoa(*port)
 	errChan := make(chan error)
 	go func() {
-		args = common.GetGCComputeArgs("ssh", host, "--", fmt.Sprintf("sudo %s --port %s --logtostderr", path.Join(testDir, cadvisorBinary), portStr))
+		args = common.GetGCComputeArgs("ssh", host, "--", fmt.Sprintf("sudo %s --port %s --logtostderr &> %s/log.txt", path.Join(testDir, cadvisorBinary), portStr, testDir))
```

Comment:
setting the log directory will result in multiple log files. I tried it locally and for some reason the INFO logs did not have all the log data. For the sake of expediency, I went with this approach. Happy to reconsider once this PR works.